src/main/java/org/sagebionetworks/web/client/Portal.java

```diff
@@ -64,62 +65,62 @@ public void onFailure(Throwable reason) {
 				@Override
 				public void onSuccess() {
 					try {
-						// start timer to check for user session state change (session expired, or user explicitly logged
-						// out)
-						ginjector.getSessionDetector().start();

```

Comment:
session/token detection requires Synapse Properties to be loaded (for endpoints)